### ISSUE TITLE
Separating JS in separate files

### ISSUE THREAD

**Issue by rehandialpad** [edgardleal/require-vuejs#32]
Is it possible to separate the js into a separate file similar to how the Vue documentation does it?

For example:
```
<template>
  <div>This will be pre-compiled</div>
</template>
<script src="./my-component.js"></script>

```
Whenever I try it, the plugin doesn't incorporate the other js.

The resulting file ends up like this:
```
(function(template){
})('' + 
' <div>' + 
' This will be pre-compiled' + 
' </div>' + 
'' + '');
```

**Comment by edgardleal** [comment]
I don't know about this feature ( src attribute in script tag ). 
I am working in this implementation and will be available soon.

**Comment by edgardleal** [comment]
This functionality is implemented on version 1.1.1.
Thank you for suggestion. 👍

**Comment by rehandialpad** [comment]
Awesome thank you!
